library/cmake/Modules/FindPHP.cmake

```diff
@@ -87,26 +91,39 @@ MARK_AS_ADVANCED(
 	PHP_VERSION_STRING
 )
 
+if(PHP_FIND_COMPONENTS)
+	foreach(component ${PHP_FIND_COMPONENTS})
+		string(TOUPPER ${component} _COMPONENT)
+		set(PHP_USE_${_COMPONENT} TRUE)
+	endforeach()
+endif()
+
+set(PHP_ZEND_COMPONENT_FOUND FALSE)
+set(PHP_ZEND_COMPONENT_ACCPETED TRUE)
```

Comment:
Why is `PHP_ZEND_COMPONENT_ACCPETED` set to `TRUE` if we are not looking for it?

Reply:
It's a bad variable name without a doubt -- basically, `PHP_ZEND_COMPONENT_FOUND` means "we found Zend", but `PHP_ZEND_COMPONENT_ACCEPTED` means "we either found Zend or we don't care".  It's a variable that's used to make `FIND_PACKAGE_HANDLE_STANDARD_ARGS()` happy, which reminds me I need to make a quick change...

Reply:
I see, according to https://cmake.org/cmake/help/v3.3/module/FindPackageHandleStandardArgs.html, all values passed to `FIND_PACKAGE_HANDLE_STANDARD_ARGS` must be valid. In case Zend is not requested, `PHP_ZEND_COMPONENT_ACCEPTED` must be `TRUE`, otherwise CMake refuses PHP although everything is fine.

Reply:
That's correct -- it's also supposed to have a `HANDLE_COMPONENTS` argument that takes care of all the components, but for some reason it's not accepting it...  So, this is the best workaround I can think of!